frontend/src/scenes/notebooks/NotebooksList/NotebooksList.tsx

```diff
@@ -105,14 +143,37 @@ export function NotebooksTable(): JSX.Element {
                 <LemonInput
                     type="search"
                     placeholder="Search for notebooks"
-                    onChange={setSearchTerm}
-                    value={searchTerm}
+                    onChange={(s) => {
+                        setFilters({ search: s })
+                    }}
+                    value={filters.search}
                 />
+                <div className="flex items-center gap-4 flex-wrap">
+                    <ContainsTypeFilters filters={filters} setFilters={setFilters} />
+                    <div className="flex items-center gap-2">
+                        <span>Created by:</span>
+                        <LemonSelect
+                            options={[
+                                { value: DEFAULT_FILTERS.createdBy as string, label: DEFAULT_FILTERS.createdBy },
+                                ...meFirstMembers.map((x) => ({
+                                    value: x.user.uuid,
+                                    label: x.user.first_name,
+                                })),
+                            ]}
+                            size="small"
+                            value={filters.createdBy}
+                            onChange={(v): void => {
+                                setFilters({ createdBy: v || DEFAULT_FILTERS.createdBy })
+                            }}
+                            dropdownMatchSelectWidth={false}
+                        />
```

Comment:
I had a very quick look at this but we have two that use UUID and two that use ID so the typing gets tricky.

I'll push this to a follow-up since my gut feeling is we shouldn't be using the ID in the other places and should change the API usage too

(but this does make four usages of this so we should probably extract it, yes 👍)